app/Http/Controllers/InitAppController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Str;
+use App\Enums\Role;
+
+class InitAppController extends Controller
+{
+    public function index()
+    {
+        if (DB::table('users')->count() > 0) {
```

Comment:
It's better to use eloquent
```suggestion
        if (User::count() > 0) {
```